Gemfile

```diff
@@ -108,6 +108,7 @@ group :test do
   gem 'factory_bot_rails', '>= 5.2.0'
   gem 'fakefs', require: 'fakefs/safe'
   gem 'faker'
+  gem 'gmail'
```

Comment:
I was curious how this Gem accesses GMail and noticed it is deprecated: https://github.com/gmailgem/gmail   The suggested replacement uses the GMail API instead and might speed processing: https://developers.google.com/gmail/api/quickstart/ruby

Not a blocker here.  Just something to think about moving to later.  Or we setup a mailcatcher-like service on another cloud where we could have ephemeral accounts and still test the full email flow.

Reply:
That's a good point! The Gmail API needs a bunch of tokens and stuff, right now we just have a simple username/password, totally open to switching, but I don't have the appetite to switch that now